### PR TITLE
chore(analytics, advanced_analitics, layers_panel, legends_panel, widgets): 12463, 13883 - merge analytics and advanced analytics panels

### PR DESCRIPTION
**Pull request by IlyaIzr**
I'll add image when adv analytics will load 😆 
Changes: 
- I've decided that layersAndLegends widget is almost the same of what I need to merge analytics, so i've created more universal widget
- Suspense wrapper added for lazy imported components -> blinking is fixed
- A simple parameter added to resizing watcher. If we don't want panel to be resized - just set it to false. Accordingly that attribute was added to panel export type
- Min height for advanced analytics was added

P.s. be not afraid of the size: lots of renaming and moving components from one place to another

### CODE REVIEW

File: src/types/featuresTypes.ts [hunk 1]
```diff
@@ -3,4 +3,7 @@ export type PanelFeatureInterface = {
   panelIcon?: JSX.Element;
   content?: JSX.Element | null;
   minHeight?: number;
+  maxHeight?: number;
+  // noShrinking?
+  skipAutoResize?: boolean;
```
Comment (IlyaIzr) [inline src/types/featuresTypes.ts:8]: should it be `noShrinking`?

**Comment by propakov** [comment]
- we have double scroll, difinetely not ok
- container resizer is broken
Screenshot 2022-12-20 at 12 52 14

File: src/features/analytics_panel/components/PanelContent/PanelContent.module.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+/* laptop */
+@media screen and (max-width: 1530px) {
+  .panelBody {
+    padding: var(--double-unit);
+    overflow-y: auto;
+  }
+}
+@media screen and (max-width: 960px) {
```
Comment (propakov) [inline src/features/analytics_panel/components/PanelContent/PanelContent.module.css:14]:
```suggestion
/* mobile */
@media screen and (max-width: 960px) {
```

**Review by Komzpa — CHANGES_REQUESTED** [review]
Double scroll is bad. 
It is ok to have resizer broken, this will get fixed with redesign of Advanced Analytics.

File: src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx [hunk 3]
```diff
@@ -67,8 +80,10 @@ export function LayersAndLegendsWidget({ layersProps, legendProps }: PanelProps)
         contentClassName={s.contentWrap}
         contentContainerRef={handleRefChange}
         customControls={panelControls}
-        contentHeight={isShort ? 'min-content' : undefined}
+        // TODO look at it
```
Comment (propakov) [inline src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx]: comment feels unclear, need more details what to do

File: src/features/advanced_analytics_panel/components/PanelContent/PanelContent.module.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.panelBody {
+  width: 790px;
+  height: 640px;
+  padding: var(--double-unit);
+  resize: both;
+  overflow: auto;
+}
+
+@media screen and (max-width: 960px) {
```
Comment (propakov) [inline src/features/advanced_analytics_panel/components/PanelContent/PanelContent.module.css:9]:
```suggestion
/* mobile */
@media screen and (max-width: 960px) {
```

File: src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx (hunk 3 above)
Comment (propakov) [reply src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx]: without details it gives nothing to person not into context

File: src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx [hunk 5]
```diff
@@ -43,15 +56,15 @@ export function LayersAndLegendsWidget({ layersProps, legendProps }: PanelProps)
 
   useAutoCollapsePanel(isOpen, onPanelClose);
 
-  if (!layersProps && !legendProps) return null;
+  if (!primaryProps && !secondaryProps) return null;
 
-  const header = !layersProps ? legendProps!.header : layersProps.header;
-  const panelIcon = !layersProps ? legendProps!.panelIcon : layersProps.panelIcon;
+  const header = !primaryProps ? secondaryProps!.header : primaryProps.header;
```
Comment (propakov) [inline src/widgets/PrimaryAndSecondaryPanel/components/PrimaryAndSecondaryPanelWidget.tsx]: I don't get the idea, why not-null assertion is used here, if type says secondaryProps and its fields are optional.
Why not just secondaryProps?.header in all places?

File: src/types/featuresTypes.ts (hunk 1 above)
Comment (propakov) [reply src/types/featuresTypes.ts:8]: it passes to `useHeightResizer` with `noResize` name, so we can reuse this one

**Comment by IlyaIzr** [comment]
Sorry, now that i've loaded adv analitics i did fixes:
mobile view:
image
desktop view (top)
image
(scrolled to bottom)
image

**Comment by propakov** [comment]
nice! 
i've just found that resizer still differs from other panels
are we going to fix it? 
Screenshot 2022-12-20 at 15 01 38

other than this resizer looks good!

**Comment by propakov** [comment]
now there is no resizer at all for advanced analytics, at least it doesn't look broken 👍 
Screenshot 2022-12-20 at 15 14 32

**Review by propakov — APPROVED** [review]
🚀

**Review by Komzpa — APPROVED** [review]

**Review by Akiyamka — APPROVED** [review]

**Pull request merged**